src/Toggle/MysqlActivatedToggleGateway.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 27/08/2015
+ * Time: 13:05
+ */
+
+namespace Clearbooks\LabsMysql\Toggle;
+
+use Clearbooks\Labs\Client\Toggle\UseCase\IsToggleActive;
+use Clearbooks\Labs\Toggle\Entity\ActivatableToggle;
+use Clearbooks\Labs\Toggle\Gateway\ActivatedToggleGateway;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Query\QueryBuilder;
+
+class MysqlActivatedToggleGateway extends MysqlGetAllTogglesGateway implements ActivatedToggleGateway
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+    /**
+     * @var IsToggleActive
+     */
+    private $toggleChecker;
+
+    /**
+     * MysqlActivatedToggleGateway constructor.
+     * @param Connection $connection
+     * @param IsToggleActive $toggleChecker
+     */
+    public function __construct( Connection $connection, IsToggleActive $toggleChecker )
+    {
+
+        $this->connection = $connection;
+        $this->toggleChecker = $toggleChecker;
+    }
+
+    /**
+     * @param string $userIdentifier
+     * @return ActivatableToggle[]
+     */
+    public function getAllMyActivatedToggles( $userIdentifier )
+    {
```

Comment:
I assume this isn't finished yet then? Because this doesn't actually get the toggles for a given user